libs/game/docs/reference/tiles/get-tile.md

```diff
@@ -1,9 +1,9 @@
-# get Tile
+# get Tile Location
```

Comment:
Name of this file should probably change to match (looks like it was already intended to from the links below)